app/src/main/java/org/dhis2/data/service/SyncDataWorker.java

```diff
@@ -116,10 +108,6 @@ public Result doWork() {
             isDataValue = false;
         }
 
-        syncController.updateDownloadProcess(false,
-                false,
-                false);
-
         syncController.finishSync();
```

Comment:
Given that you have injected `SyncStatusController` on `SynPresenterImpl`, you could move this call to the presenter and remove the injection of `SyncStatuscontroller` from `SyncDataWorker`. What do you think?